opentelemetry-api/src/opentelemetry/distributedcontext/__init__.py

```diff
@@ -11,3 +11,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+
+# TODO
+class DistributedContext:
```

Comment:
If this stays a single-file package, we should consider making it a module (i.e., move distributedcontext/__init__.py to distributedcontext.py)

Reply:
I agree, but I picked this up from #68. This PR is blocked on that one, I just put it up so others could build the docs from this branch.

In any case I think @lzchen is going to save distributed context for another PR.